pkg/metrics/proxy/injector/pod_patch.go

```diff
@@ -60,6 +60,14 @@ func (i *injector) getPodPatchOperations(
 	// the proxy sidecar and proxy init container
 	portMappingStr := strings.Join(portMappingStrs, ",")
 
+	// Build the list of "ignored paths" that won't be counted by appending the
+	// custom list from the pod to the global list from the config
+	ignoredPaths := strings.Split(i.config.IgnoredPaths, ",")
+	ignoredPaths = append(
+		ignoredPaths,
+		kubernetes.GetIgnoredPaths(pod.Annotations)...,
+	)
+
```

Comment:
I'm perplexed about how this is meant to work. It looks like this is meant to find the values later used in setting the `IGNORED_PATHS` env var, but it's retrieving some of that from the pod's ignored paths annotation... and I can't see how that annotation ever got set.

Reply:
Nevermind. I see how this gets set now.